scripts/download_metamask.ts

```diff
@@ -0,0 +1,13 @@
+import downloader, {defaultDirectory} from "../src/setup/utils/metaMaskDownloader";
+import {RECOMMENDED_METAMASK_VERSION} from "../src"
+
+(async function() {
+    await downloader(RECOMMENDED_METAMASK_VERSION, {
+        location: defaultDirectory,
+        flask: false,
+      });
+    await downloader(RECOMMENDED_METAMASK_VERSION, {
+        location: defaultDirectory,
+        flask: true,
+      });
```

Comment:
`location` is an optional param, seen a change in `src/setup/utils/metaMaskDownloader.ts` on `defaultDirectory` which is used here. Do not see a reason for it as not providing will use it.